mention-bot.js

```diff
@@ -221,6 +227,25 @@ function getSortedOwners(
   return sorted_owners;
 }
 
+function getDefaultOwners(
+  files: Array<FileInfo>,
+  whitelist: Array<WhitelistUser>
+): Array<string> {
+  var owners = [];
+
+  whitelist.forEach(function(user) {
+    var userHasChangedFile = files.find(function(file) {
+      return minimatch(file.path, user.files);
+    });
+
+    if (userHasChangedFile) {
+      owners.push(user.name);
```

Comment:
you may push the same user multiple times, you need to uniq-ify that list at the end :)